pip/commands/install.py

```diff
@@ -178,6 +179,10 @@ def __init__(self, *args, **kw):
     def run(self, options, args):
         cmdoptions.check_install_build_global(options)
 
+        upgrade_strategy = "not-allowed"
```

Comment:
Uhm... What would be the right verb/term for the current behavior without `--upgrade` then? "to-satisfy-only"?